ArduCopter/commands.cpp

```diff
@@ -21,34 +21,38 @@ void Copter::update_home_from_EKF()
 void Copter::set_home_to_current_location_inflight() {
     // get current location from EKF
     Location temp_loc;
-    if (inertial_nav.get_location(temp_loc)) {
-        const struct Location &ekf_origin = inertial_nav.get_origin();
-        temp_loc.alt = ekf_origin.alt;
-        if (!set_home(temp_loc, false)) {
-            return;
-        }
+    if (!ahrs.get_location(temp_loc)) {
+        return;
+    }
+    Location ekf_origin;
+    if (!ahrs.get_origin(ekf_origin)) {
+        return;
+    }
+    temp_loc.alt = ekf_origin.alt;
+    if (!set_home(temp_loc, false)) {
+        return;
+    }
         // we have successfully set AHRS home, set it for SmartRTL
```

Comment:
Re-indent the comment please. And moving it to inside the macro is probably a good idea.